packages/discord.js/src/structures/BaseGuildTextChannel.js

```diff
@@ -160,34 +175,48 @@ class BaseGuildTextChannel extends GuildChannel {
    *   .then(invite => console.log(`Created an invite with a code of ${invite.code}`))
    *   .catch(console.error);
    */
-  createInvite(options) {
+  async createInvite(options) {
     return this.guild.invites.create(this.id, options);
   }
 
   /**
    * Fetches a collection of invites to this guild channel.
    * Resolves with a collection mapping invites by their codes.
+   *
    * @param {boolean} [cache=true] Whether or not to cache the fetched invites
    * @returns {Promise<Collection<string, Invite>>}
    */
-  fetchInvites(cache = true) {
+  async fetchInvites(cache = true) {
     return this.guild.invites.fetch({ channelId: this.id, cache });
   }
 
   // These are here only for documentation purposes - they are implemented by TextBasedChannel
-  /* eslint-disable no-empty-function */
+
+  /* eslint-disable getter-return */
   get lastMessage() {}
+
   get lastPinAt() {}
+
   send() {}
+
   sendTyping() {}
+
   createMessageCollector() {}
+
   awaitMessages() {}
+
   createMessageComponentCollector() {}
+
   awaitMessageComponent() {}
+
   bulkDelete() {}
+
   fetchWebhooks() {}
+
   createWebhook() {}
+
   setRateLimitPerUser() {}
+
   setNSFW() {}
```

Comment:
I've seen other methods like this modified to have async. Should these have it too?